pkgs/development/python-modules/stanza/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib,
+buildPythonPackage,
+python3Packages,
+pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "stanza";
+  version = "1.3.0";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "1wqhz82p158vhch3z3vglgy0s8qwr2aj83dm6y5qgkpbximc3l3h";
+  };
+
+  disabled = pythonOlder "3.6";
+
+  propagatedBuildInputs = with python3Packages; [ 
+    emoji
+    numpy
+    protobuf
+    requests
+    six
+    pytorch
+    tqdm
+  ];
+
+  doCheck = false;
```

Comment:
To run the tests, a shell script is called in `stanza`. But this is being changed to a python script in the next release (see [Issue 820](https://github.com/stanfordnlp/stanza/pull/820). Should I wait until then?

And please explain to me under which circumstances `doCheck = false;` is allowed, since I am not understanding how to run the tests correctly. Right now the error I get while running tests goes as the following (stemming from the shell script:

```
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /stanfordnlp/stanza-resources/main/resources_1.3.0.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fffa2c5a790>: Failed to establish a new connection: [Errno -2] Name or service not known'))
```

I did fix up the rest of your comments and changed it to `fetchFromGitHub` as source. Will push later on.

Reply:
The tests try to connect to the internet which is forbidden in the sandbox. Either disable the tests that do that or disable them entirely. Either way this needs that information as a comment and a pythonImportsChecks.

Reply:
Pushed again to the branch. Please have another look, thanks!